src/main/java/com/beerjournal/breweriana/user/User.java

```diff
@@ -30,4 +30,8 @@ public static User of(String firstName, String lastName, String email, String pa
         return new User(null, firstName, lastName, email, password);
     }
 
+    public static User ofModifiable(ObjectId id, User user) {
```

Comment:
Isn't this name misleading? Class User is definitely not modifiable. Maybe just "of" or "copyOfWithId". I see that this class is not in persistence package so you can make this method package private. For consistency you can move other classes (Item, UserCollection etc) out of persistence package